DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -574,6 +574,15 @@ private void PostUIInit()
         /// </summary>
         private void ConnectionManager_BannedFromChannel(object sender, ChannelEventArgs e)
         {
+            var chatChannel = connectionManager.FindChannel(e.ChannelName);
+            if (chatChannel != null)
+            {
+                if(chatChannel.ChannelName != ProgramConstants.GeneralCncnetChatChannelName)
```

Comment:
The purpose of the overall IF statement is to show the user that it's a specific chat channel that they're banned from, for one, not cncnet as a whole, which is what everyone is thinking when they see `#cncnet` in the error message.

This specific IF statement for the general chat can be removed in the future, if/when the ban is removed from the general chat in IRC.